src/ergw_aaa_session.erl

```diff
@@ -316,14 +309,10 @@ handle_event(info, {'$reply', Promise, Handler, Msg, Opts} = _Info,
     aaa_state_stats(Handler, State, StateOut),
     Data = Data0#data{handlers = maps:put(Handler, StateOut, HandlersS), session = SessOut},
     update_session(SessOut, EvsOut, Data),
-    {keep_state, Data};
-
-handle_event(Type, Event, _State, _Data) ->
-    ?LOG(warning, "unhandled event ~p:~p", [Type, Event]),
-    keep_state_and_data.
+    {keep_state, Data}.
```

Comment:
Maybe keep as notice?

Reply:
there should not be any unexpected event in production code

Reply:
Famous last words XD

Reply:
But ok, unhandled events would not have a catchall claus and a crash would be visible. Fine for me.